waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml

```diff
@@ -2301,4 +2301,16 @@
         </sql>
     </changeSet>
 
+
+    <!-- 212 control rendering of static panels -->
+    <changeSet author="kamransaleem" id="20160805-212-1">
```

Comment:
I don't think this should be a database change.  It's up to the designer to choose at render time how the panels would best fit in their page.